Develop

lst-gg opened the pull request.

lst-gg committed:
add operations

lst-gg committed:
fix Wallet is not a valid constructor

lst-gg committed:
minor fixes

llamas-laptrinh commented on example/util/common.tsx:
Please use a `.ts` extension instead of a `.tsx` extension when you are not exporting a UI

lst-gg replied:
done

llamas-laptrinh commented on example/util/getPoolBidDetails.tsx:
Please use a .ts extension instead of a .tsx extension when you are not exporting a UI

llamas-laptrinh commented:
Thank you for contributing to this repo, I'm happy to see your PR and I will merge it after you update some of the things I commented and add what you did to the readme file.

llamas-laptrinh commented on example/util/getPoolBidDetails.tsx:
```diff
@@ -0,0 +1,154 @@
+  }
+
+  // retrieve initial highest bid price if pool is double sided (config.startingPrice would be the initial lowest list price in that case)
+  var startingPriceBidSide: Big;
```
In most cases, prefer let to var unless the variable needs to be declared globally." This provides more context and suggests let as the default choice.

lst-gg committed:
add api key

lst-gg committed:
move API KEY to .env

lst-gg committed:
Fix ts files

lst-gg committed:
fix .env api key

lst-gg committed:
try connect get collection

lst-gg committed:
get collections

lst-gg committed:
Changes to be committed:
	modified:   example/app/page.tsx
	modified:   example/components/MagicEdenLoginButton/index.tsx
	modified:   example/util/getCollections.ts

Dev

lst-gg commented:
> Thank you for contributing to this repo, I'm happy to see your PR and I will merge it after you update some of the things I commented and add what you did to the readme file.

Hi, could you please check the updated PR

lst-gg committed:
Fix get collections on Magic Eden
